com/win32com/client/build.py

```diff
@@ -631,8 +628,8 @@ def _BuildArgList(fdesc, names):
     while len(names) < numArgs:
         names.append("arg%d" % (len(names),))
     # As per BuildCallList(), avoid huge lines.
-    # Hack a "\n" at the end of every 5th name - "strides" would be handy
-    # here but don't exist in 2.2
+    # Hack a "\n" at the end of every 5th name
+    # TODO: "strides" would be handy here but don't exist in 2.2
     for i in range(0, len(names), 5):
         names[i] = names[i] + "\n\t\t\t"
```

Comment:
What strides are we talking about? I can only find references to `memoryview.strides` and `numpy.ndarray.strides`. Is this comment still relevant?

Reply:
No, I think this comment can be killed.